### PR TITLE
Update portuguese PT-BR Language

### PR DESCRIPTION
**Pull request by BrnoNebuu**
more and more sugestions by changing some portuguese errors

ID: 23839113

email: gometrydash23@gmail.com

### CODE REVIEW

File: res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -929,9 +929,9 @@
     <string name="error_1">Apenas o líder ou o vice-líder pode enviar mensagens para todos do clã.</string>
     <string name="Hat">Boné</string>
     <string name="Hats">Bonés</string>
-    <string name="Pet_Uploads">Envios de Mascote</string>
+    <string name="Pet_Uploads">Envios de Mascotes</string>
     <string name="BLOB">BLOB</string>
-    <string name="Clan_Pet_Uploads">Envios de Masc. Clã</string>
+    <string name="Clan_Pet_Uploads">Envios de Mascotes do Clã</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:934]: Please never undo the abbreviations. They are made for a good reason.

File: res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -976,13 +976,13 @@ Se você notar um jogador quebrando as regras do jogo, denuncie a um Moderador.
     <string name="OPEN">ABRIR</string>
     <string name="Halo">Aréola</string>
     <string name="To_use_this_halo_you_must_be_at_least_level_s_and_unlock_s_achievements">"Para usar esta aréola, você deve ter pelo menos o nível  %1$s e ganhar %2$s conquistas."</string>
-    <string name="Halos">Aréolas</string>
+    <string name="Halos">Auréolas</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:979]: Both "aréolas" and "auréolas" are already correct. Do not change it, as people are already used to "aréolas". You can even make a research of it for proof.

File: res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -976,13 +976,13 @@ Se você notar um jogador quebrando as regras do jogo, denuncie a um Moderador.
...
     <string name="CAMPAIGN">CAMPANHA</string>
     <string name="Mission">Missão</string>
     <string name="Kill_All_Bots">Mate Todos os Bots</string>
     <string name="Survive">Sobreviva</string>
     <string name="Mission_d_Complete">Missão %d Completa!</string>
-    <string name="Mission_d_Failed">Missão %d Falida!</string>
+    <string name="Mission_d_Failed">Missão %d Falhou!</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:985]: "Falida" was already correct, even though it sounds more "informal".

File: res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -1010,15 +1010,15 @@ Se você notar um jogador quebrando as regras do jogo, denuncie a um Moderador.
     <string name="PROFILE_COLOR">COR DO PERFIL</string>
     <string name="Enable_Colors">Ativar Cores</string>
     <string name="Particles">Partículas</string>
-    <string name="Clan_Particle_Uploads">Envios de Partíc. Clã</string>
+    <string name="Clan_Particle_Uploads">Envios de Partículas do Clã</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:1013]: Same issue of line 934.

File: res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -1126,10 +1126,10 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
     <string name="SKIN_PACK">PACOTE DE SKIN</string>
     <string name="HALLOWEEN">DIA DAS BRUXAS</string>
     <string name="pack_info">Depois que você comprar este pacote de skin, suas skins exclusivas aparecerão na parte superior dos menus de seleção de skin.</string>
-    <string name="EXCLUSIVE_skins_pets_hats_and_more_">Skins, Masc, Bonés EXCLUSIVOS e Mais!</string>
+    <string name="EXCLUSIVE_skins_pets_hats_and_more_">Skins, Masc, Bonés EXCLUSIVOS e Muito Mais!</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:1129]: Previous text was already correct (based on english value).

File: res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -1126,10 +1126,10 @@ Esta é uma lista de coisas que são *PROIBIDAS* para usar nas skins personaliza
...
     <string name="BUY_NOW_">COMPRAR AGORA!</string>
     <string name="already_purchased">Você ou a pessoa para qual você está comprando já possui isto.</string>
-    <string name="Halo_Animations">Animações da Aréola</string>
+    <string name="Halo_Animations">Animações da Auréola</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:1132]: Same issue of line 979.

File: res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -1360,13 +1360,13 @@ Comandos:
     <string name="Master_Tamer">Domador Magistral</string>
     <string name="Get_ALL_Pets_To_Level_100_">Leve TODOS os mascotes ao nível 100</string>
     <string name="TAMER">DOMADOR</string>
-    <string name="Allow_Recombine">Permitir Recombinar</string>
+    <string name="Allow_Recombine">Permitir Recombinaçaõ</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:1363]: 2 issues:

1. Your change contains a typo;
2. The previous text was already correct (based on english value).

File: res/values-pt-rBR/strings.xml [hunk 8]
```diff
@@ -1397,20 +1397,20 @@ Comandos:
...
+- Enviar skins criadas por usuários.
 
 Se você não desejar fazer login, ainda poderá jogar no modo único jogador."
     </string>
     <string name="_2x_skin_resolution">Qualidade da Skin em Dobro</string>
     <string name="pm">PRIV.</string>
-    <string name="You_are_already_friends_">Você já tem esta pessoa adicionada.</string>
-    <string name="Account_must_be_1_day_old_to_send_mail_">Esta conta deve ter pelo menos 1 dia de criação para enviar mensagens.</string>
+    <string name="You_are_already_friends_">Você já tem essa pessoa adicionada.</string>
+    <string name="Account_must_be_1_day_old_to_send_mail_">A conta deve ter pelo menos 1 dia de criação para enviar mensagens.</string>
```
Comment (Jeii20) [inline res/values-pt-rBR/strings.xml:1413]: Line 1412: Previous text was already correct.
Line 1413: "Esta" was already correct for better interpretation.

**Comment by Jeii20** [comment]
PR closed. Too many invalid changes in the mentioned lines.

**Comment by BrnoNebuu** [comment]
1 - I search in the google the diference of aérola and auréola, and appears
this:

Unlike what is stated in the album “Bizarre or curious tattoos”, the
correct one is 'areola(aérola)' - which means “circle, of color containing,
around the nipple”, according to the Michaelis dictionary -, and not
'halo(auréola)' - “any luminous circle that surrounds an object” -,
according to the same glossary.

The "aureula" sounds more logic for the game. So, why i cant change the
aérola to auréola? Why changed the Battle Royale to Batalha Real?? This
doesnt makes sence, Because the braziliam community is acustumed with this.

2 - I Litteraly checked the chat gpt to see if have a portuguese errors,
qnd the Falhou; Recombinação; "A conta...", everything here is correct, to
make the language more formal and easy to understand.

3 - I doenst know about the abreviations, thanks to inform me.

Pls reply to this comment.

Em seg, 16 de set de 2024 14:20, Jeii20 ***@***.***> escreveu:

**Comment by Jeii20** [comment]
The "recombinação" part you are right, but it's not written like "recombination" to be the reason of that change. Exactly, that's what you read about the "halo" one: whatever if it's "aréola" or "auréola", you've seen right now that both are correct and there's no any need to switch it to other if their meanings are the same. Remind that you don't need to be totally formal everywhere, even the most popular games and socials aren't completely formal. "Battle Royale" is not a commonly used feature and almost nobody plays it, if it wasn't because of that factor, I wouldn't change it. @BrnoNebuu

**Comment by BrnoNebuu** [comment]
What changes i do its correct?

Em seg, 16 de set de 2024 17:11, Jeii20 ***@***.***> escreveu:

**Comment by BrnoNebuu** [comment]
1 - the recombine and recombination part, you is correct, but i tried to do
a better "text".

2 - This is the thing i noticed, the formal language its not a actual
language today

3 - We preffer Battle Royale (because of free fire).

Em seg, 16 de set de 2024 22:04, Geometry Dash ***@***.***>
escreveu:

**Comment by BrnoNebuu** [comment]
Please reply both messages 😊

Em seg, 16 de set de 2024 22:12, Geometry Dash ***@***.***>
escreveu:

**Comment by Jeii20** [comment]
@BrnoNebuu a good suggestion for the change of "Falida" is "Falhada". ^^

**Comment by BrnoNebuu** [comment]
Add me on discord for easy communication

brunonebuu700

Em ter, 17 de set de 2024 12:51, Geometry Dash ***@***.***>
escreveu:

**Pull request closed without merging**